content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,229 @@
+        controller</a> (<a href="/docs/notebooks/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/pipelines">Pipelines</a>
+          (<a href="/docs/pipelines/pipelines-quickstart/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">Profile 
+        Controller</a> (<a href="/docs/other-guides/multi-user-overview/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/pytorch-operator">PyTorch operator</a> (<a href="/docs/components/training/pytorch/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td>PyTorch serving 
+          (<a href="/docs/components/serving/pytorchserving/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/batch-predict">TensorFlow 
+          batch prediction</a>
+          (<a href="/docs/components/serving/tfbatchpredict/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>TensorFlow serving (<a href="/docs/components/serving/tfserving_new/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/tf-operator">TFJob operator</a>
+          (<a href="/docs/components/training/tftraining/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/xgboost-operator">XGBoost operator</a></td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="sdk-matrix"></a>
+## Kubeflow SDKs and CLIs
+
+Alongside Kubeflow {{% kf-latest-version %}}, you may want to use 
+one of the following Kubeflow SDKs and command-line interfaces 
+(CLIs).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>SDK / CLI</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
+        <th>SDK/CLI version</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://github.com/kubeflow/fairing">Fairing</a> (<a href="/docs/fairing/reference/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kfctl">kfctl</a> (<a href="/docs/other-guides/kustomize/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://kubeflow-pipelines.readthedocs.io/en/latest/">Kubeflow Pipelines SDK</a> (<a href="/docs/pipelines/sdk/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
```

Comment:
Should `kustomize` be in this table? I don't fully understand its relationship to kfctl (maybe the kfctl row is sufficient) but I see you added a "stable" notice to https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/other-guides/kustomize/ in this PR

Reply:
That's a good question. The "stable" notice on that doc refers to kfctl rather than kustomize. (The doc started out as kustomize only, but then it made more sense to combine the kfctl and kustomize instructions as they're so intertwined.)

My understanding is that kustomize is kind of outside if Kubeflow, as it's something people can download and use if they need to tweak their config, but it's not deployed in their Kubeflow cluster. 

So, we could say the same for kfctl - people need to download kfctl, and it's not deployed to their cluster. Plus, I guess there's a kustomize component in kfctl in order to make the initial setup work. But I think it makes sense to treat kustomize as a dependency rather than a component of Kubeflow. kfctl isn't a component either, but it's listed below as an SDK/CLI, because it's a core part of Kubeflow.

If we treat kustomize purely as a dependency, then it's enough to specify the minimum version as we do here:
https://www.kubeflow.org/docs/other-guides/kustomize/#installing-kustomize

Though I think we could specify the min version in more places, so I've raised https://github.com/kubeflow/website/issues/1438

Does that sound good?

Reply:
Yes, thanks for the explanation! I think you can resolve this thread. (Is there no way for me to resolve a comment as a reviewer?)

Reply:
Thanks! I don't know about reviewers resolving the conversation, but I have the power!